web/templates/confirm_dialog/confirm_reset_stream_notifications.hbs

```diff
@@ -0,0 +1,6 @@
+<p>
+    {{#tr}}
+    Are you sure you want to reset notifications for <z-stream></z-stream> ?
+    {{#*inline "z-stream"}}{{> ../inline_decorated_stream_name stream=sub show_colored_icon=true}}{{/inline}}
```

Comment:
I think we should not show colored icon here. We just show black icons in at least one more modal I guess.